p2panda-rs/src/operation/operation_fields.rs

```diff
@@ -120,12 +121,25 @@ impl Validate for OperationFields {
     }
 }
 
+impl TryFrom<Vec<(&str, OperationValue)>> for OperationFields {
```

Comment:
In the refactor this is not required anymore as the `OperationBuilder` has a similar API (it takes in `.fields(&[(&str, OperationValue)])`)